chain-abstraction/interfaces.go

```diff
@@ -415,5 +423,5 @@ type SpecEdge interface {
 		prefixProof []byte,
 	) (VerifiedRoyalEdge, VerifiedRoyalEdge, error)
 	// Confirms an edge for having a total timer >= one challenge period.
-	ConfirmByTimer(ctx context.Context) (*types.Transaction, error)
+	ConfirmByTimer(ctx context.Context, claimedAssertion AssertionHash) (*types.Transaction, error)
```

Comment:
Should be on verified royal, because it gives us confidence that it is honest. We wouldn't want to accidentally call ConfirmByTimer on an edge that is not honest, although that wouldn't happen. I'll add a doc comment